### PR TITLE
chore(release): Release for Fuse Ignite GA

### PR DESCRIPTION
**Pull request by dsimansk**
I've preset what is expected to be GA version to prevent any confusion. And also because of the fact that plugin seems to be ignoring cmd line property like `-Dsyndesis.version`.

I'd like to contribute this extension into `syndesis-extensions` to be used with upstream version.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -18,7 +18,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>io.syndesis.example</groupId>
   <artifactId>sample-damage-reporter-extension</artifactId>
-  <version>1.3-SNAPSHOT</version>
+  <version>1.3.10</version>
```
Comment (zregvart) [inline pom.xml]: Do we want the version to be `1.3.10` I thought we always wanted it to be `-SNAPSHOT` on `master`?

File: pom.xml [hunk 2]
```diff
@@ -33,7 +33,7 @@
     <maven-compiler-plugin.version>3.7.0</maven-compiler-plugin.version>
 
     <!-- dependencies -->
-    <syndesis.version>1.3-SNAPSHOT</syndesis.version>
+    <syndesis.version>1.3.10.fuse-000001-redhat-1</syndesis.version>
```
Comment (zregvart) [inline pom.xml:36]: This is fine, but it adds a bit of a maintenance, perhaps we could remove it and enforce that the `syndesis.version` must be specified with `-D`?

**Review by zregvart — COMMENTED** [review]
Two little issues, perhaps not that important.

File: pom.xml (hunk 1 above)
Comment (dsimansk) [reply pom.xml]: Sure, I'll revert it.

File: pom.xml (hunk 2 above)
Comment (dsimansk) [reply pom.xml:36]: Would you mind if we keep it simple and address it later? :)

Comment (zregvart) [reply pom.xml:36]: Not at all.

File: pom.xml (hunk 1 above)
Comment (dsimansk) [reply pom.xml]: @zregvart this was addressed. Feel free to approve :)

**Comment by dsimansk** [comment]
@zregvart could we get this merged pretty pls? :)

**Review by zregvart — APPROVED** [review]
:+1:

**Pull request merged**